volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -152,9 +290,173 @@ def get_subkeys(self) -> Iterable[interfaces.objects.ObjectInterface]:
             subkey_node = hive.get_cell(self.SubKeyLists[index]).u.KeyIndex
             yield from self._get_subkeys_recursive(hive, subkey_node)
 
+    def get_int_value(
+        self, name: str, default: Optional[T] = None
```

Comment:
The idea behind using `T` here instead of int was that it gives consumers the ability to decide what type they want to return in the case of missing values. I definitely didn't want `renderers` this deep in the code, so this was my attempt to solve that same problem again.